core/nmf-composites/nanosat-mo-supervisor/src/main/java/esa/mo/nmf/nanosatmosupervisor/parameter/DummyValuesProvider.java

```diff
@@ -33,13 +35,19 @@
  */
 public class DummyValuesProvider extends OBSWParameterValuesProvider {
 
+  /**
+   * Dummy Map of OBSW parameter value by parameter name acting as our cache storage.
+   */
+  private final Map<Identifier, Attribute> cache;
```

Comment:
I would rename it - not really a cache per se.
Rather `parameterStorage`, `valuesStore`, `parameterStore`, `simulatedParamStore`, etc.